sqlglot/dialects/dialect.py

```diff
@@ -184,17 +184,17 @@ class Dialect(metaclass=_Dialect):
     """Determines how function names are going to be normalized."""
 
     LOG_BASE_FIRST = True
-    """Determines whether the base comes first in the `LOG` function."""
+    """Determines whether or not the base comes first in the `LOG` function."""
```

Comment:
PyCharm's default inspector complains about "whether or not", saying the "or not" is redundant haha

Reply:
Lol, kinda makes sense.. I'll get rid of it.